.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Tests
+
+on:
+  pull_request: {}
+  push:
+    branches: [master]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version:
+        # - "3.6"
```

Comment:
We aren't running tests for Python 3.6, but `setup.py` says this package supports it.

Reply:
Yeah, `actions/setup-python` no longer supports 3.6 in an x64 arch. I might start another PR to officially drop support for 3.6. Wdyt?

Reply:
Seeing as Meltano doesn't support Python 3.6 either, I don't mind dropping it from `target-sqlite`.

Reply:
Agreed: b5a0765188a322dc21988c7aae0944fb03a7d7cb